pkgs/applications/misc/bottles/default.nix

```diff
@@ -20,8 +20,8 @@ let
 in
 python3Packages.buildPythonApplication rec {
   pname = "bottles";
-  version = "2022.4.14-trento";
-  sha256 = "0kjc1w8x4d6g2lx8x8isa2vnwacyjlh9lldf14wn4b118hsw85zs";
+  version = "2022.4.14-trento-1";
```

Comment:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=bottles#n37

Matches with the current version in AUR (that is "official" as far I understood from the website).